mxd/3.1_MXD_tutorial1_dataexchange.md

```diff
@@ -0,0 +1,181 @@
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "protocol": "dataspace-protocol-http",
+    "counterPartyAddress": "http://bob-controlplane:8084/api/v1/dsp",
+    "querySpec": {
+        "offset": 0,
+        "limit": 100,
+        "filter": "",
+        "range": {
+            "from": 0,
+            "to": 100
+        },
+        "criterion": ""
+    }
+}'
+```
+
+Let´s see if Alice can see the Asset (ID:3). Can you find it?
+
+As you can see in the response, the data offer "Product EDC Demo Asset 3" (asset ID:3) does not appear. Unfortunately, Alice sees some contract offers but she cannot find the contract offer from Bob.
+
+Alice calls Bob and says she can´t see the asset. Bob remembers that he did not create an access policy. An access policy defines who is allowed to see a data offering.
+
+Action (Bob): Create the access policy using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/policydefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "3",
+  "policy": {
+    "@type": "Policy"
+  }
+}' 
+```
+
+Bob tells Alice that he has created the right policy. Let´s see if Alice can now find the data asset. Execute the request again using the following curl command:
+
+```shell
+curl --location 'http://localhost/alice/management/v2/catalog/request' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "protocol": "dataspace-protocol-http",
+    "counterPartyAddress": "http://bob-controlplane:8084/api/v1/dsp",
+    "querySpec": {
+        "offset": 0,
+        "limit": 100,
+        "filter": "",
+        "range": {
+            "from": 0,
+            "to": 100
+        },
+        "criterion": ""
+    }
+}'
+```
+
+Let´s see if Alice can see the Asset. Can you find it?
+
+Once again Alice cannot find the data offer. This is by design and to be expected since Bob has only created an asset and a policy definition. An asset and a policy cannot be displayed to Alice as a consumer without a contract definition.
+**This is the first lesson for this tutorial: A contract must be defined between two parties that want to exchange data. This contract must always contain an asset and a policy.**
+
+![Contract Definition](assets/components_contract_definition.png)
+
+Action (Bob): Create a contract definition including the asset and the policy you have created. For this, use the following curl command:
+
+```shell
+This is the correct curl
+
+curl --location 'http://localhost/bob/management/v2/contractdefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {},
+  "@id": "3",
+  "@type": "ContractDefinition",
+  "accessPolicyId": "3",
```

Comment:
it is important to understand, and illustrate, that access policy and contract policy are two **distinct** policies, that serve separate purposes. it is bad practice to use the same policy for both!!

Reply:
@hemantxpatel

Reply:
@jkbquabeck  create a contract policy just above this step.
```
```shell
curl --location 'http://localhost/bob/management/v2/policydefinitions' \
--header 'Content-Type: application/json' \
--header 'X-Api-Key: password' \
--data-raw '{
  "@context": {
    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "PolicyDefinitionRequestDto",
  "@id": "32",
  "policy": {
    "@type": "Policy"
  }
}' 
```
```